### PR TITLE
Embed manifest to the golang binary instead download it

### PR DESCRIPTION
**Pull request by gabrielbussolo**
# Summary of this proposal

## Issue(s) number(s)

*Disclaimer: Please do not create a Pull Request without creating an issue first.*

## Checklist for submitter

- [ ] My PR has a related issue/bug number.
- [ ] My PR provides tests
  - [ ] Integration tests are added/passes
  - [ ] Unit tests are added/passes
  - [ ] End-to-end tests added/passes
  - [ ] Static code check added/passes
  - [ ] Linting on documentation added/passes
  - [ ] Does not affect code coverage stats
- [ ] My PR requires updating dependencies
- [ ] My PR has the documentation changes required.

## Checklist for reviewer

- [ ] The proposal fixes a bug/issue or implements a new feature that is well described.
- [ ] The proposal has sufficient test cases that covers the changes.
  - [ ] If changes an API, it does not break backwards compatibility (-1 major version).
- [ ] If integration is required, the proposal has integration tests.
- [ ] The proposal does not break coverage stats
- [ ] The proposal has the required documentation changes.

## Backport

- [ ] Backport to the latest stable release.

### CODE REVIEW

File: cmd/operator/manifest.yaml
Comment (patrick-stephens) [inline cmd/operator/manifest.yaml:1]: Where does this file end up on disk or is it inside the binary?

Comment (gabrielbussolo) [reply cmd/operator/manifest.yaml:1]: binary, its embedded

**Comment by patrick-stephens** [comment]
Needs a rebase I think.

File: cmd/operator/install.go [hunk 1]
```diff
@@ -22,8 +23,10 @@ import (
 	"strings"
 )
 
+//go:embed manifest.yaml
```
Comment (jasmingacic) [inline cmd/operator/install.go:26]: @patrick-stephens it gets embedded here by go generate

**Review by jasmingacic — COMMENTED** [review]
This is one way of doing it. But I'd really recommend that we embed whole config/default so we can have a little bit more flexibility. At the end of the day the result should be the same but with a single manifest we had to manually intervene and change references to service account and what not. With the approach in #499 we could drop that part.

**Pull request closed without merging**